### PR TITLE
Update documentation - fix typo error and fix broken link

### LINKED ISSUE #515: Fix all wrong typos and broken link in documents
### Checklist
- [x] Bug from Service (Vatz)
- [ ] Bug from Plugin (Vatz) 
- [ ] Malfunction on CI/CD 
- [x] etc(e.g, Hardware version doesn't match, )

---

There are still typo error on README.md and contains broken links

1. ReadME.md has typos error
2. ReadME.md has a broken link 

image

- https://github.com/dsrvlabs/vatz/blob/main/docs/community_plugins.md

### PR DESCRIPTION
**Pull request by xellos00**
### 1. Type of change

Please delete options that are not relevant.

- [ ] New feature
- [ ] Enhancement
- [ ] Bug/fix (non-breaking change which fixes an issue)
- [x] others (anything other than above)

---

close #515

**Summary**

Update fix broken link, typos, and transfer community plugin instruction to https://github.com/dsrvlabs/vatz-community-plugins

---

### CODE REVIEW

**Review by meetrick — APPROVED** [review]
LGTM

File: manager/dispatcher/discord.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package dispatcher
 import (
 	"bytes"
 	"encoding/json"
+	"errors"
```
Comment (heejin-github) [inline manager/dispatcher/discord.go:6]: It seems not related all changes in `manager` with this PR and issue

**Review by heejin-github — CHANGES_REQUESTED** [review]

File: manager/dispatcher/discord.go (hunk 1 above)
Comment (xellos00) [reply manager/dispatcher/discord.go]: Agree, I removed all the changes that's been committed with PR https://github.com/dsrvlabs/vatz/pull/514.
Can you check again?

**Review by rootwarp — APPROVED** [review]
LGTM

**Review by heejin-github — APPROVED** [review]
LGTM

File: docs/community_plugins.md [hunk 2]
```diff
@@ -1,32 +1,8 @@
 # Community Plugins
 
-Share your custom plugin with others by adding it to the list using templete below! Check out the [contribution guide](https://github.com/dsrvlabs/vatz/blob/main/docs/contributing.md) and PR to update. (PR will be reviewed by DSRV)
-You could remove it via PR, but only if the person who added it to the list requests to be removed from the list.
+Share your own plugins with everyone by adding to the list at [vatz-community-plugins](https://github.com/dsrvlabs/vatz-community-plugins) <br>
+Other contribution policies are availble at [contribution guide](https://github.com/dsrvlabs/vatz/blob/main/docs/contributing.md).
```
Comment (qkrybin) [inline docs/community_plugins.md:4]: availble -> available

Comment (xellos00) [reply docs/community_plugins.md]: @qkrybin 
Updated!! plz, review again.

**Review by qkrybin — APPROVED** [review]
LGTM

**Pull request merged**